### PR TITLE
feat: add footer and feature components

### PR DESCRIPTION
**Pull request by apoorvcodes**

### CODE REVIEW

File: src/components/Features.vue [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<template>
+ <div>
+	o
```
Comment (savioxavier) [inline src/components/Features.vue]: There's a random `o` here lol

File: src/pages/index.vue [hunk 2]
```diff
@@ -2,15 +2,15 @@
   <section class="min-w-screen min-h-screen">
     <WhatsNew />
     <Hero />
-    <WhyMinima/>
```
Comment (savioxavier) [inline src/pages/index.vue:5]: Honestly, why was this component removed? You could have kept it, or maybe even moved it to a separate page.

**Comment by megatank58** [comment]
Axios is one of the slowest performing http request libraries which doesn't make it a great candidate here. Undici is one of the fastest due to its many optimizations. In addition, it provides many spec compliant structures like FormData that are cross compatible with the web standards. (And it'll be built into node soon) So we should replace axios with undici here.

**Comment by savioxavier** [comment]
> Axios is one of the slowest performing http request libraries which doesn't make it a great candidate here. Undici is one of the fastest due to its many optimizations. In addition, it provides many spec compliant structures like FormData that are cross compatible with the web standards. (And it'll be built into node soon) So we should replace axios with undici here.

Isn't Axios one of most popular libraries out there, used by millions? At least it's better than node-fetch. Unidici's barely starting out. I think we should stick with Axios, I don't see why we should switch considering it works just fine.

**Comment by megatank58** [comment]
> > Axios is one of the slowest performing http request libraries which doesn't make it a great candidate here. Undici is one of the fastest due to its many optimizations. In addition, it provides many spec compliant structures like FormData that are cross compatible with the web standards. (And it'll be built into node soon) So we should replace axios with undici here.
> 
> Isn't Axios one of most popular libraries out there, used by millions? At least it's better than node-fetch. Unidici's barely starting out. I think we should stick with Axios, I don't see why we should switch considering it works just fine. 

Axios downloads: 25,416,438
node-fetch downloads: 35,972,655

Undici is the way to go as it is going to be the standard in node v18 and it'll be built-in, we also don't need any fancy features yet so it's just future proof. Also how's axios better than node-fetch :thinking:

**Comment by apoorvcodes** [comment]
I'll be switching it Undici, tbh that what the plan was but for quick testing I just used axios

**Comment by apoorvcodes** [comment]
@megatank58 This is the rough new site there are some fixes needed like mobile stuff which will be done by @savioxavier I believe

**Comment by savioxavier** [comment]
> @megatank58 This is the rough new site there are some fixes needed like mobile stuff which will be done by @savioxavier I believe

Needs a bit of code cleanup, component responsiveness, text and button scaling and wording fixes. Looks fine for me now, as I'll be fixing them in a later PR

**Comment by savioxavier** [comment]
Squash and merge this btw, @megatank58. Too many commits.

**Comment by apoorvcodes** [comment]
> https://deploy-preview-10--gominima.netlify.app



> Squash and merge this btw, @megatank58. Too many commits.

Haha ikr did a lot of fixes

**Comment by savioxavier** [comment]
@apoorvcodes was `axios` replaced with `undici`, as per megatank's comment?

**Comment by apoorvcodes** [comment]
> @apoorvcodes was `axios` replaced with `undici`, as per megatank's comment?

I didn't mega will do it

**Comment by megatank58** [comment]
> Squash and merge this btw, @megatank58. Too many commits. 

All of the coming merges on any of our repositories (with the exception of /docs) will be squashed

**Comment by savioxavier** [comment]
> Axios downloads: 25,416,438
> node-fetch downloads: 35,972,655

Having more downloads doesn't necessarily mean it's more popular among developers. I just suggested Axios because I've used it over node-fetch in most of my projects. A lot of developers prefer Axios over built-in APIs for its ease of use.

> Undici is the way to go as it is going to be the standard in node v18 and it'll be built-in, we also don't need any fancy features yet so it's just future proof.

I didn't see the point of switching to another library, since the only web request is a simple GitHub API fetch, which isn't too complex for axios to handle. However, if Undici seems like a better choice, if more API requests are to be added to the site in the future, and http optimization is key, then go for it!

>  Also how's axios better than node-fetch 🤔

I could cite a bunch of articles regarding this, but I don't think it's necessary.

We'll go with Undeci, since you both seem to agree with it. I just wanted to clarify my point here.

**Comment by megatank58** [comment]
> > Axios downloads: 25,416,438
> > node-fetch downloads: 35,972,655
> 
> Having more downloads doesn't necessarily mean it's more popular among developers. I just suggested Axios because I've used it over node-fetch in most of my projects. A lot of developers prefer Axios over built-in APIs for its ease of use.

I suggested node-fetch mainly because I'm more familiar with it and it's used in discord.js, while i do agree more downloads is not more popularity, that's a different topic.

> > Undici is the way to go as it is going to be the standard in node v18 and it'll be built-in, we also don't need any fancy features yet so it's just future proof.
> 
> I didn't see the point of switching to another library, since the only web request is a simple GitHub API fetch, which isn't too complex for axios to handle. However, if Undici seems like a better choice, if more API requests are to be added to the site in the future, and http optimization is key, then go for it!

The docs are rendered by fetching JSON which contains the documentation which will grow a lot in size as time goes own with multiple JSON for multiple projects and their own multiple branches and versions

**Review by savioxavier — APPROVED** [review]
I kinda like the new Dracula dark theme, nice work, Apoorv

Waiting for @megatank58 to review and merge.

Also, check if the PR title is appropriate for the merge, please.

File: src/pages/index.vue (hunk 2 above)
Comment (apoorvcodes) [reply src/pages/index.vue:5]: > Honestly, why was this component removed? You could have kept it, or maybe even moved it to a separate page.

It was poorly made due to the reason removing it was the best option

Comment (savioxavier) [reply src/pages/index.vue:5]: > It was poorly made due to the reason removing it was the best option

A wise choice. The website looks so much better now. 👍

Comment (apoorvcodes) [reply src/pages/index.vue:5]: > 

Ofc it would because its made by a frontend expert while giving up on his sleep

**Pull request merged**